Thumbnail/LiipImagineThumbnail.php

```diff
@@ -35,16 +35,20 @@ public function __construct(RouterInterface $router)
      */
     public function generatePublicUrl(MediaProviderInterface $provider, MediaInterface $media, $format)
     {
-        if ($format == 'reference') {
-            $path = $provider->getReferenceImage($media);
-        } else {
-            $path = $this->router->generate(
-                sprintf('_imagine_%s', $format),
-                array('path' => sprintf('%s/%s_%s.jpg', $provider->generatePath($media), $media->getId(), $format))
-            );
+        $path = $provider->getReferenceImage($media);
+
+        if ('admin' === $format || 'reference' === $format) {
+            return $path;
         }
 
-        return $provider->getCdnPath($path, $media->getCdnIsFlushable());
+        $path = $provider->getCdnPath($path, $media->getCdnIsFlushable());
+
+        $params = array(
+            'path' => ltrim($path, '/'),
+            'filter' => $format,
+        );
+
+        return $this->router->generate('liip_imagine_filter', $params, UrlGeneratorInterface::ABSOLUTE_URL);
```

Comment:
We are misusing suggest section in some of our bundles:

https://getcomposer.org/doc/04-schema.md#suggest

It is suposed to have a description of what does that bundle (liipimagine) if installed.
Can you change to provide a description?